src/test/app/TxQ_test.cpp

```diff
@@ -2943,7 +2943,7 @@ class TxQ1_test : public beast::unit_test::suite
         // may not reduce to 8.
         env.close();
         checkMetrics(__LINE__, env, 9, 50, 6, 5, 256);
-        BEAST_EXPECT(env.seq(alice) == aliceSeq + 13);
```

Comment:
Not sure why this test was failing or why this fixes it.

Reply:
This test is fragile and dependent on ordering of transactions, which is affected by the closed/validated ledger hash. It wouldn't hurt to add a comment to that effect. Since this PR changes the ledger structure, this fix is fine. It may need a rewrite to make it future-proof, but I haven't figured out how to do that.